packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -648,6 +648,12 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin {
     final LogicalKeyboardKey key = keyEvent.logicalKey;
 
     final bool isMacOS = keyEvent.data is RawKeyEventDataMacOs;
+
+    // Ignores CapsLock if it's a Backspace key event. On Android the backspace
+    // key on the "extra symbols" keypad also holds down Capslock.
+    // See: https://github.com/flutter/flutter/issues/75616.
```

Comment:
That issue link is a 404.

Also: Really?  That's very strange behavior. Perhaps we should limit removing it to the Android platform.

/cc @dkwingsmt I think perhaps you should be aware of this in regards to lock tracking.

Reply:
The link works for me (there's a period at the end that's not part of the url).

Reply:
<kbd>backspace</kbd> seems to be the only key that's sent as a key event on that keyboard.

Reply:
According to the keyboard's keymapping files, the <kbd>*#</kbd> turns shift-lock on and the keymapping changes (the num keys are replaced with symbols) based on the shift lock state. 

> Perhaps we should limit removing it to the Android platform

Isn't this behavior universal? Pressing backspace with capslock on works in X11 too.

Reply:
Ahh, it's just GitHub's link detector that is including the "."

Actually I think this is just an instance of this bug, where we're deciding that the key is down when the lock is on. https://github.com/flutter/flutter/issues/64761 @dkwingsmt's work separates those concepts, so it should be fixed by that, and then this PR won't be necessary (I hope).

Reply:
Ah sounds good. I'm gonna close this PR and mark the issue as a duplicate of #64761.